setup.sh

```diff
@@ -13,10 +13,10 @@ function install_once {
     python-smbus python-pip i2c-tools \
     libxml2 python-dev python-pycurl python-pyquery \
     imagemagick
-  run sudo pip install --upgrade pip
-  run sudo pip install --upgrade \
+  run sudo -H pip install --upgrade pip
```

Comment:
pip and/or npm has been complaining to me that the .cache directory is owned by the wrong user.